docs/charts_tips_and_tricks.md

```diff
@@ -96,6 +96,36 @@ For example:
 The above will render the template when .Values.foo is defined, but will fail
 to render and exit when .Values.foo is undefined.
 
+## Creating Image Pull Secrets
+Image pull secrets are essentially a combination of _registry_, _username_, and _password_.  You may need them in an application you are deploying, but to create them requires running base64 a couple of times.  A helper can help you do this.  Here is an example:
+
+Presuming that the values are provided in the values.yaml like so
```

Comment:
- [ ] s/Presuming that the values are provided in the values.yaml like so/First, assume that the credentials are defined in the _values.yaml_ file like so:/

Reply:
We usually write file and directory names in italic style, not the monospace face.